### DISCUSSION TITLE
LXQt 2.0 articles

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2555] [Media]
https://twitter.com/lxqt_project/status/1780579645756223525

https://mastodon.social/@LXQt/112286637653040400

https://www.phoronix.com/news/LXQt-2.0-Released

https://9to5linux.com/lxqt-2-0-desktop-environment-officially-released-this-is-whats-new

https://alternativeto.net/news/2024/4/lxqt-2-0-launches-with-qt-6-6-improved-wayland-support-and-a-new-fancy-menu-/

https://linuxiac.com/lxqt-2-0-desktop-environment/

https://linuxnews.de/lxqt-2-0-setzt-auf-qt-6-und-wayland/

https://www.muylinux.com/2024/04/17/lxqt-2-0-qt6/

**Comment by stefonarch** [comment]
@yan12125 when are arch packages coming?

**Reply by yan12125** [reply]
I expect after a few weeks

**Reply by stefonarch** [reply]
For LXQt 2.0? What are the issues?

**Reply by yan12125** [reply]
> What are the issues?

I just created a list about works to do: https://archlinux.org/todo/lxqt-20/

**Reply by yan12125** [reply]
An update about LXQt 2.0 status in Arch Linux: an Arch Linux developer suggests (in a non-public Matrix room for Arch Linux Package Maintainers) waiting for Python 3.12 rebuilds [1][2] before starting LXQt 2.0. I have modified PKGBUILDs locally, but updated packages are not pushed yet for the reason above.

[1] https://archlinux.org/todo/python-modules-part-1/
[2] https://archlinux.org/todo/python-312-part-2/

**Reply by stefonarch** [reply]
I don't get the connection  between LXQt and Python, but I think you do ;)

**Reply by yan12125** [reply]
Fortunately, no new issues in non-LXQt packages, and everything is moved to testing repos :)

> I don't get the connection between LXQt and Python, but I think you do ;)

It was just a precaution - Python affects tons of packages, and moving along with it might be tricky.

**Reply by stefonarch** [reply]
>Fortunately, no new issues in non-LXQt packages, and everything is moved to testing repos :)

I see, there we go :)

https://archlinux.org/packages/?sort=&q=lxqt-session&maintainer=&flagged=

Thanks!

**Comment by stefonarch** [comment]
New entry, more journalism-like:
https://www.theregister.com/2024/04/20/lxqt2_updates_to_qt6/
